parts/gearbox.go

```diff
@@ -0,0 +1,45 @@
+package parts
+
+import "fmt"
+
+type Gearbox struct {
```

Comment:
This is one way to have the gearbox manipulate the wheels based on motor changes, but it is not adhering to the note in Go. #5.d from the README.md file:

`d) NOTE: A "Car" will have a gearbox as a component. All other components should not interact without the car itself. IE - Gearbox does not directly manipulate wheels. Motor does not directly manipulate gearbox. All linkage between components happens at the "Car" level.`